Gemfile

```diff
@@ -70,40 +144,40 @@ gem 'mail', :git => 'https://github.com/mikel/mail', :branch => '2-6-stable'
 
 # Assets
 gem 'bootstrap-sass', '~> 2.3.2.2'
-gem 'sass-rails', '~> 5.0.5'
-gem 'compass-rails', '3.0.2'
-gem 'coffee-rails', '~> 4.0.1'
-gem 'uglifier', '~> 2.7.2'
-gem 'therubyracer', '~> 0.12.2'
+gem 'sass-rails', '~> 5.0.0'
+gem 'compass-rails', '~> 3.0.0'
+gem 'coffee-rails', '~> 4.0.0'
+gem 'uglifier', '~> 2.7.0'
+gem 'therubyracer', '~> 0.12.0'
 
 # Feature flags
 gem 'alaveteli_features', :path => 'gems/alaveteli_features'
 
 group :test do
   gem 'fakeweb', '~> 1.3.0'
-  gem 'coveralls', '< 0.8.5', :require => false
-    gem 'tins', '< 1.3.1'
-    gem 'term-ansicolor', '< 1.4'
+  gem 'coveralls', '~> 0.8.0', :require => false
+    gem 'tins', '~> 1.3.0', '< 1.3.1'
+    gem 'term-ansicolor', '~> 1.3.0', '< 1.4'
   gem 'capybara', '~> 2.7.0'
   gem 'delorean', '~> 2.1.0'
 end
 
 group :test, :development do
   gem 'bullet', '~> 5.1.0'
   gem 'factory_girl_rails', '~> 4.7.0'
-  gem 'rspec-activemodel-mocks', '~> 1.0.1'
+  gem 'rspec-activemodel-mocks', '~> 1.0.0'
   gem 'rspec-rails', '~> 3.4.0'
-  gem 'pry', '~> 0.10.4'
-  gem 'pry-debugger', '~> 0.2.3', :platforms => :ruby_19
-    gem 'public_suffix', '< 1.5.0'
+  gem 'pry', '~> 0.10.0'
+  gem 'pry-debugger', '~> 0.2.0', :platforms => :ruby_19
+    gem 'public_suffix', '~> 1.4.0', '< 1.5.0'
 end
```

Comment:
I think we can finally add this now:

`gem 'pry-byebug', '~> 3.4.0', :platforms => [:ruby_20, :ruby_21]`

\o/